docker/transformers-pytorch-tpu/bert-base-cased.jsonnet

```diff
@@ -0,0 +1,38 @@
+local base = import 'templates/base.libsonnet';
+local tpus = import 'templates/tpus.libsonnet';
+local utils = import "templates/utils.libsonnet";
+local volumes = import "templates/volumes.libsonnet";
+
+local bertBasedCase = base.BaseTest {
+  frameworkPrefix: "hf",
+  modelName: "bert-based-case",
```

Comment:
Shouldn't the `bert-based-case` and `bertBasedCase` be `bert-base-cased` and `bertBaseCased`?

By the way, why is this model specific? Are we using these values somewhere?

Reply:
Fixed that typo.

The `local bertBasedCase` is just a name of that jsonnet variable, it just needs to be a unique string but several punctuation marks are not allowed in variable names. We recommend 1 variable like this per test. On our team we have 1 file per "family" of tests, e.g. 1 file for pytorch resnet50 where the file contains [short test, long test] X [v2-8 TPU, v3-8 TPU, v100 GPU(s)] for the same model on the same ML framework. 

The `modelName` and `frameworkPrefix` are just used in generating the name of the GKE job. For example, a recent run was named: `hf-bert-based-case-example-v3-8-vtgrj`. Future runs will look like `hf-bert-base-cased-example-v3-8-xxxxx`